src/data/cloud-computing-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const cloudComputingQuiz = [
+  {
+    Question:
+      "What does the scalability benefit of cloud-native apps enable?",
+    Answer: "Flexible deployment options across the network",
+    Distractor1: "Rigid and fixed deployment options",
+    Distractor2: "Limited development and deployment options",
+    Distractor3: "Static and unchangeable application structure",
+    Explanation:
+      "Cloud-native apps allow flexible deployment options across the network, enabling developers to scale and adapt their applications as needed, making them more resilient and responsive to changing demands.",
```

Comment:
```suggestion
      "Scalability is a benefit because it allows cloud native applications to have flexible deployment options across the network making it easier to develop, deploy, and iterate on the application.",
```